besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/RocksDbHelper.java

```diff
@@ -66,36 +67,213 @@ static void forEachColumnFamily(
+      final String cf_file_histogram = "cf-file-histogram";
+      final String cf_write_stall_stats = "cf-write-stall-stats";
+      final String dbstats = "dbstats";
+      final String db_write_stall_stats = "db-write-stall-stats";
+      final String levelstats = "levelstats";
+      final String block_cache_entry_stats = "block-cache-entry-stats";
+      final String fast_block_cache_entry_stats = "fast-block-cache-entry-stats";
+      final String num_immutable_mem_table = "num-immutable-mem-table";
+      final String num_immutable_mem_table_flushed = "num-immutable-mem-table-flushed";
+      final String mem_table_flush_pending = "mem-table-flush-pending";
+      final String compaction_pending = "compaction-pending";
+      final String background_errors = "background-errors";
+      final String cur_size_active_mem_table = "cur-size-active-mem-table";
+      final String cur_size_all_mem_tables = "cur-size-all-mem-tables";
+      final String size_all_mem_tables = "size-all-mem-tables";
+      final String num_entries_active_mem_table = "num-entries-active-mem-table";
+      final String num_entries_imm_mem_tables = "num-entries-imm-mem-tables";
+      final String num_deletes_active_mem_table = "num-deletes-active-mem-table";
+      final String num_deletes_imm_mem_tables = "num-deletes-imm-mem-tables";
+      final String estimate_num_keys = "estimate-num-keys";
+      final String estimate_table_readers_mem = "estimate-table-readers-mem";
+      final String is_file_deletions_enabled = "is-file-deletions-enabled";
+      final String num_snapshots = "num-snapshots";
+      final String oldest_snapshot_time = "oldest-snapshot-time";
+      final String oldest_snapshot_sequence = "oldest-snapshot-sequence";
+      final String num_live_versions = "num-live-versions";
+      final String current_version_number = "current-super-version-number";
+      final String estimate_live_data_size = "estimate-live-data-size";
+      final String min_log_number_to_keep_str = "min-log-number-to-keep";
+      final String min_obsolete_sst_number_to_keep_str = "min-obsolete-sst-number-to-keep";
+      final String base_level_str = "base-level";
+      final String total_sst_files_size = "total-sst-files-size";
+      final String live_sst_files_size = "live-sst-files-size";
+      final String obsolete_sst_files_size = "obsolete-sst-files-size";
+      final String live_sst_files_size_at_temperature = "live-sst-files-size-at-temperature";
+      final String estimate_pending_comp_bytes = "estimate-pending-compaction-bytes";
+      final String aggregated_table_properties = "aggregated-table-properties";
+      final String num_running_compactions = "num-running-compactions";
+      final String num_running_flushes = "num-running-flushes";
+      final String actual_delayed_write_rate = "actual-delayed-write-rate";
+      final String is_write_stopped = "is-write-stopped";
+      final String estimate_oldest_key_time = "estimate-oldest-key-time";
```

Comment:
It looks like this metric is not available when compaction is enabled, which is the case in Besu.